### ISSUE TITLE
RR lost frames drops to 0

### ISSUE THREAD

**Issue by wowaser** [ultravideo/uvgRTP#154]
Hello, 

I have a hook for RTCP receiver reports in my program. Here is a simplified version:
```cpp
void rtcpHandler::receiverRTCPHook(std::shared_ptr<uvgrtp::frame::rtcp_receiver_report> report)
{
	for (auto& block : report->report_blocks) {
		uint32_t lost = block.lost;
		std::cout << "rtcp report lost: " << lost << "\n";
	}
}
```

I introduce packet loss into my program with clumsy (https://jagt.github.io/clumsy/download.html)

If I keep running the program with consistent packet loss for a couple of minutes, the **block.lost** becomes 0 and never recovers, regardless of packet loss situation. I have the latest uvg version, but this bug has been there for quite a while now
![image](https://user-images.githubusercontent.com/70014877/180418230-1c314d4f-154e-41cc-89dc-482b233cd79c.png)

**Comment by wowaser** [comment]
I found the bug, I will make a pull request at some point

**Comment by wowaser** [comment]
Solved by #156
